_FIPS201/authentication.md

```diff
@@ -314,16 +317,16 @@ Some characteristics of the visual authentication mechanism include the followin
 ## 6.3 PIV Support of Graduated Authenticator Assurance Levels {#s-6-3}
 
 The PIV Card supports a set of authentication mechanisms that can be used to implement graduated
-assurance levels. The assurance levels used within this Standard are closely aligned with
+assurance levels. The assurance levels used for remote/networked access within this Standard are closely aligned with
 NIST [[SP 800-63]](../_Appendix/references.md#ref-SP-800-63), which specifies a digital identity risk management process that is cited by OMB [[M-19-17]](../_Appendix/references.md#ref-OMB1917).
 
 The following subsections specify which PIV
-authentication mechanisms **CAN** be used to support the various authenticator assurance levels
+authentication mechanisms can be used to support the various identity assurance levels
```

Comment:
But this should be authenticator assurance levels, right?

Reply:
```suggestion
authentication mechanisms can be used to support the various authenticator assurance levels
```

Reply:
```suggestion
authentication mechanisms can be used to support the various assurance levels
```